### DISCUSSION TITLE
Can Wazuh Agents connect over an IP other than the Management interface?

### DISCUSSION THREAD

**Discussion by HelenH321** [Security-Onion-Solutions/securityonion#6873] [General] [answered]
I have installed a new standalone so-manager (2.3.70 - planning to soup it to 2.3.91 next week), but our design calls for the Wazuh agents to talk to it on a different network than the management interface.  The plan had been to do this over the other NIC, but on investigation that NIC appears to be more of a sniffing type monitor network than an agent monitoring network.

Can anyone advise if this is possible/has anyone had success at setting it up this way before, or do I need to revise the design to get the Wazuh agents talking on the management network?

Thanks

**Comment by dougburks** [comment, marked as answer]
If you have a standalone manager with two NICs, then the management NIC has an IP address but the other NIC does not as it is just for passively sniffing network traffic. We recommend configuring Wazuh agents to talk to to the IP address of the management NIC.
